src/App.js

```diff
@@ -9,7 +9,7 @@ const App = () => (
     <Navbar />
     <Routes>
       <Route index element={<Navigate to="/books" replace />} />
-      <Route path="/books" element={<Books />} />
+      <Route path="/books" element={<Books className="books-container" />} />
       <Route path="/categories" element={<Categories />} />
     </Routes>
   </div>
```

Comment:
- [ ] Your Styling is as expected however you should remove the spaces between the pages  cause you should only take the page in the design not all on the image literally also it doesn't make sense for a website like that so please try to remove in order to complete the requirements
![Screenshot 2023-03-01 at 15 06 58](https://user-images.githubusercontent.com/61790587/222164591-b0fc5bfb-476c-44f0-bef2-7387e55470e7.png)